js/components/controlBar.js

```diff
@@ -502,14 +519,19 @@ var ControlBar = React.createClass({
     this.moreOptionsItems = collapsedMoreOptionsItems;
 
     finalControlBarItems = [];
-
+     
     for (var k = 0; k < collapsedControlBarItems.length; k++) {
       if (collapsedControlBarItems[k].name === "moreOptions" && (this.props.controller.state.isOoyalaAds || collapsedMoreOptionsItems.length === 0)) {
         continue;
       }
-
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
-    }
+      var position = 'center';
+      if (k === collapsedControlBarItems.length -1) {
```

Comment:
position is the tooltips 'pointer' alignment, in retrospect this variable should've been named alignment. Thanks!